README.md

```diff
@@ -35,8 +38,28 @@ You will also require a trello api key and api token. In order to generate these
 More information about generation api keys and tokens for trello can be found [here](https://developer.atlassian.com/cloud/trello/guides/rest-api/api-introduction/)
 
 
-## Virtual environment setup
+## Running the app using docker
+
+There is a multi-stage docker file for this project, containing a production build and a development build:
+
+### Production:
+- To produce the image: `docker build --target production --tag todo-app:prod .`
+- To run: `docker run -p 5100:80 -d --env-file .env todo-app:prod`
+
+The app will then run be accessible on localhost:5100.
+
+### Development: 
+
+The development container can be launched using `docker-compose up -d`. The app can then be accessed at localhost:5000 and will reload whenever changes are made to the app files locally.
```

Comment:
FYI this will not use the latest version of the dockerfile unless you include the `--build` option.

Reply:
Ah, thanks, good to know 🙂